### ISSUE TITLE
"No module named pfTICLProducer_cfi" in wf 20493.52

### ISSUE THREAD

**Issue by silviodonato** [cms-sw/cmssw#29446]
After https://github.com/cms-sw/cmssw/pull/29081, we are getting the following error in 20493.52 since CMSSW_11_1_X_2020-04-09-1100
```
Traceback (most recent call last):
  File "/cvmfs/cms-ib.cern.ch/nweek-02623/slc7_aarch64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-04-09-1100/bin/slc7_aarch64_gcc820/cmsDriver.py", line 56, in &lt;module&gt;
    run()
  File "/cvmfs/cms-ib.cern.ch/nweek-02623/slc7_aarch64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-04-09-1100/bin/slc7_aarch64_gcc820/cmsDriver.py", line 28, in run
    configBuilder.prepare()
  File "/cvmfs/cms-ib.cern.ch/nweek-02623/slc7_aarch64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-04-09-1100/python/Configuration/Applications/ConfigBuilder.py", line 2266, in prepare
    self.pythonCfgCode += self.addCustomise()
  File "/cvmfs/cms-ib.cern.ch/nweek-02623/slc7_aarch64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-04-09-1100/python/Configuration/Applications/ConfigBuilder.py", line 866, in addCustomise
    __import__(packageName)
  File "/cvmfs/cms-ib.cern.ch/nweek-02623/slc7_aarch64_gcc820/cms/cmssw/CMSSW_11_1_X_2020-04-09-1100/python/RecoHGCal/TICL/ticl_iterations.py", line 19, in &lt;module&gt;
    from RecoHGCal.TICL.pfTICLProducer_cfi import pfTICLProducer
ImportError: No module named pfTICLProducer_cfi
```
@rovere @felicepantaleo

**Comment by silviodonato** [comment]
assign reconstruction

**Comment by silviodonato** [comment]
assign upgrade

**Comment by perrotta** [comment]
I think the automatic replacement of *Producer in the product names as applied in the last commits of  #29081 included by mistake the replacement pointed out in https://github.com/cms-sw/cmssw/pull/29081/files#r406646968

I think it is eough to revert that line change to fix the issue

**Comment by perrotta** [comment]
I have created #29450 with a fix

**Comment by rovere** [comment]
> I have created #29450 with a fix

@perrotta I'll have a look and provide a fix asap.

**Comment by rovere** [comment]
@silviodonato @perrotta @kpedro88 @slava77 
The failing workflows are the ones that were customized to run TICL.
Now that TICL is part of the official reconstruction, there's no need to have them as separate workflows.
I see 2 options:

1. Remove the workflows from runTheMatrix
2. Leave the workflows and remove the old customization.

I'd be in favour of 1, but I'll wait for input from your side before proceeding.

**Comment by silviodonato** [comment]
I agree to remove 20493.52, since now we are running TICL also in many other workflows.

**Comment by perrotta** [comment]
> I agree to remove 20493.52, since now we are running TICL also in many other workflows.

I also agree

**Comment by rovere** [comment]
Great!
I proposed a patch in #29540, simply because @perrotta had it already open.

**Comment by silviodonato** [comment]
Fixed by #29450
